README.md

```diff
@@ -2,18 +2,38 @@
 
 This service is used to predict which tests need to run based on code changes.
 
-# Set up environment
+## Making API requests
+Because the selected-tests service is hosted in Kanopy, authentication is done
+through CorpSecure. To make requests to the selected-tests API you will need to
+include your CorpSecure auth_user and auth_token cookies in your request.
+
+To make an API request, follow the following steps:
+1. Log into the selected-tests service by going to the following link in your
+   browser and logging in:
+   https://selected-tests.server-tig.prod.corp.mongodb.com/health
+2. Get your personal auth_token and auth_user cookies for the
+   https://selected-tests.server-tig.prod.corp.mongodb.com domain. (You can find
+   these under the Application tab in Chrome console.)
```

Comment:
I think "under the Application tab" tab needs to be fleshed out. Not many backend developers have used any web console.

A screenshot highlighting the "Cookies" item on the sidebar and highlighting the "auth_token" and "auth_user" cookies in the list of many cookies would be nice.

I would also say that it's expected for the page to return a 404

Reply:
👍 Added a screenshot and explained unauthenticated requests will result in a 302